manifests/g/GeodeSDK/GeodeCLI/2.7.1/GeodeSDK.GeodeCLI.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: GeodeSDK.GeodeCLI
+PackageVersion: 2.7.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: geode.exe
+  PortableCommandAlias: geode

```

Comment:
```suggestion
  PortableCommandAlias: geode
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
VCRedist is needed for the application to run successfully
<!--
[Policy] Dependencies
-->